api/src/main/java/marquez/service/models/Job.java

```diff
@@ -41,6 +41,9 @@ public final class Job {
   @Nullable @Setter private Run latestRun;
   @Getter private final ImmutableMap<String, Object> facets;
   @Nullable private UUID currentVersion;
+  @Getter @Nullable private String jobType;
```

Comment:
Is `jobType` and `processingType` type the same?

Reply:
Since `Job.Type` is already defined, having both `Job.Type` and `Job.JobType` would be very confusing (specifically on usage). I don't think Marquez needs to be one-to-one in naming with OpenLineage (though I think we should certainly try).

First, I would map `processingType` to the [`JobType`](https://github.com/MarquezProject/marquez/blob/main/api/src/main/java/marquez/common/models/JobType.java) defined by Marquez. And, I _think_ we can view `Job.JobType` as a sub type of `Job.Type`. That is, a `Job.Type` of `BATCH` can be a `DAG` or `TASK`. I also view `integration` as a qualifier of the `Job`. So, I'd favor having both the `JobType` and `integration` defined by OpenLineage as `labels`  as mentioned in my earlier [comment](https://github.com/MarquezProject/marquez/pull/2682#discussion_r1408311828)